javascript-timers/interval.js

```diff
@@ -0,0 +1,18 @@
+var $countdown = document.querySelector('.countdown-display');
+function countdownDisplay() {
+  var string = $countdown.textContent;
+  var int = parseInt(string);
```

Comment:
This makes it so that your code is dependent on values on the DOM in order to know what to do, and that is bad. You should make a variable to hold the count and use that to know what the current count is rather than pulling values from DOM elements.